examples/feature_columns.R

```diff
@@ -0,0 +1,84 @@
+library(tensorflow)
+library(tfdatasets)
+library(tfestimators)
+library(keras)
+library(tidyverse)
+
+URL = 'https://storage.googleapis.com/applied-dl/heart.csv'
```

Comment:
@dfalbel we should copy this file to the package to avoid this getting broken when lacking internet connectivity or if this file gets removed, specially since the file is really small.